### DISCUSSION TITLE
[ODIN] add preset/modifiers

### DISCUSSION THREAD

**Discussion by Encrypt01** [mbround18/valheim-docker#738] [Ideas]
Can you add an odin argument for the new modifier/preset pretty please with a cloudberry on top?
![valupdate](https://github.com/mbround18/valheim-docker/assets/34519392/b9c94664-9eef-419d-999a-8b8798462dee)

**Comment by mbround18** [comment]
@Encrypt01 you can do this now via `SERVER_EXTRA_LAUNCH_ARGS` :) but i can add dedicated env vars

**Reply by mbround18** [reply]
@Encrypt01 This is in progress #742

**Reply by cyrrelsneer** [reply]
Hey @mbround18 , thanks so much for that.
I am not sure on how to use SERVER_EXTRA_LAUNCH_ARGS - could you give an example maybe?
Highly appreciated

**Reply by mbround18** [reply]
![image](https://github.com/mbround18/valheim-docker/assets/12646562/84b40b34-a6b3-4ad4-8eca-574655e6388a)

**Reply by mbround18** [reply]
new version im going to release has these built in now

| Variable                  | Default           | Required | Description                                                                                                                                                                                                                                                                                                                     |
| ------------------------- | ----------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| PRESET                    | ``                | FALSE    | Normal, Casual, Easy, Hard, Hardcore, Immersive, Hammer                                                                                                                                                                                                                                                                         |
| MODIFIERS                 | ``                | FALSE    | Comma-separated array of modifiers. EX: `combat=easy,raids=muchmore`                                                                                                                                                                                                                                                            |
| SET_KEY                   | ``                | FALSE    | Can be one of the following: nobuildcost, playerevents, passivemobs, nomap                                                                                                                                                                                                                                                      |

**Reply by mbround18** [reply]
merged in https://github.com/mbround18/valheim-docker/releases/tag/v2.3.0
